frontend/src/_layouts/base.html

```diff
@@ -1,118 +1,161 @@
-              <span class="mobile-s:block mobile-s:whitespace-nowrap sm:inline">The Official Archive of</span>
-              <span class="mobile-s:whitespace-nowrap">His Divine Grace</span>
-            </span>
-            <span class="bg-lotus hidden w-7 flex-none -scale-x-100 bg-contain bg-right-top bg-no-repeat sm:block"></span>
-          </span>
-          <span class="lg:text-xl">
-            <span class="whitespace-nowrap">Sri Srimad</span>
-            <span class="mobile-s:block mobile-s:whitespace-nowrap sm:inline">Bhaktivedanta Narayana Gosvami Maharaja</span>
-          </span>
-        </div>
-        <img
-          src="~/src/images/gurudeva.png"
-          alt="Srila Gurudeva"
-          class="absolute left-0 right-0 top-full z-10 mx-auto w-28 -translate-y-3 rounded-full sm:left-4 sm:right-auto md:left-20 lg:left-20 lg:w-44 lg:-translate-y-16"
-        >
-      </div>
-    </header>
-    <nav class="relative mb-4 mt-0.5 bg-submarine-500 p-4 sm:mb-0 sm:p-0">
-      <!-- Mobile menu button -->
-      <div class="flex justify-end sm:hidden">
-        <button type="button" data-menu-toggle class="p-2 text-teal-800 focus:outline-none lg:hidden" aria-controls="mobile-menu" aria-expanded="false">
-          <span class="sr-only">Open main menu</span>
-          <!-- Icon when menu is closed. -->
-          <!-- Heroicon name: outline/menu -->
-          <svg data-menu-toggled class="block h-6 w-6" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor" aria-hidden="true">
-            <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 6h16M4 12h16M4 18h16" />
-          </svg>
-          <!-- Icon when menu is open. -->
-          <!-- Heroicon name: outline/x -->
-          <svg data-menu-toggled class="hidden h-6 w-6" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor" aria-hidden="true">
-            <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M6 18L18 6M6 6l12 12" />
-          </svg>
-        </button>
-      </div>
-      <!-- Menu items -->
-      <div data-menu-toggled class="hidden sm:block">
-        <div class="flex flex-col items-end justify-end space-y-4 py-1 pl-36 font-header text-sm font-semibold text-gray-600 child-a:px-2 child-a:py-1 sm:flex-row sm:items-stretch sm:justify-start sm:space-x-1 sm:space-y-0 md:pl-56 lg:justify-center lg:space-x-3 lg:pl-72 lg:pr-28 lg:text-base">
-          {%- assign menu = collections.topnav | eleventyNavigation -%}
-          {%- for entry in menu -%}
-            <a
-              href="{{ entry.url }}"
-              {% if entry.url == page.url %}
-                class="rounded-md bg-teal-800 text-white"
-              {% endif %}
-              {% if entry.target == 'external' %}
-                target="_blank"
-              {% endif %}
-            >
-              {{ entry.title }}
-            </a>
-          {%- endfor -%}
+  <!doctype html>
+  <html>
```

Comment:
I've read somewhere that it's preferable to apply style to the `body` tag rather than to the `html` one. There is subtle functional difference but more important is the semantic meaning. See, previously we had classes on the `body`.